axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceMergingController.java

```diff
@@ -0,0 +1,201 @@
+package com.axelor.apps.account.web;
+
+import com.axelor.apps.account.db.Invoice;
+import com.axelor.apps.account.db.PaymentCondition;
+import com.axelor.apps.account.db.PaymentMode;
+import com.axelor.apps.account.db.repo.InvoiceRepository;
+import com.axelor.apps.account.service.invoice.InvoiceMergingService;
+import com.axelor.apps.account.service.invoice.InvoiceMergingService.InvoiceMergingResult;
+import com.axelor.apps.base.db.Partner;
+import com.axelor.apps.base.db.PriceList;
+import com.axelor.apps.base.db.Wizard;
+import com.axelor.apps.tool.MapTools;
+import com.axelor.auth.db.AuditableModel;
+import com.axelor.exception.service.TraceBackService;
+import com.axelor.inject.Beans;
+import com.axelor.meta.schema.actions.ActionView;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import org.apache.commons.collections.CollectionUtils;
+
+public class InvoiceMergingController {
+
+  private String getViewName(InvoiceMergingResult result) {
```

Comment:
Please change this method name so it is more specific ( `getMergeConfirmFormViewName` ?)